packages/xstate-react/src/useInterpret.ts

```diff
@@ -128,7 +140,8 @@ export function useInterpret<
     Object.assign(service.machine.options.activities, activities);
     Object.assign(service.machine.options.services, services);
     Object.assign(service.machine.options.delays, delays);
-  }, [actions, guards, activities, services, delays]);
+    Object.assign(service.machine.options.input, input);
+  }, [actions, guards, activities, services, delays, input]);
```

Comment:
Could you point me to a test that would fail if we'd remove this dependency from here (and the associated `Object.assign` call)? It would help me to understand this better